From f2ac354f79eb3ffc237cee9d67d4131ab88ee1ad Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Mon, 27 Jun 2016 11:42:29 +0000
Subject: [PATCH 206/462] Mantis 0041425 extension added SSDM Application

fixed path to json jar
added build path ${PlatformJavaDir}/SSDMApplication

git-svn-id: svn://localhost/SelfServiceBuild/trunk@314 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 4 ++--
 SelfServiceBuild/Build/java/build.xml         | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index aed2536..d051f2a 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -178,7 +178,7 @@
 <property name="JarServlet"                 value="${DIR_GLOBALS}/ThirdParty/servlet-api/0.1/jar/servlet.jar"/>
 <property name="JarSNMP"                    value="${DIR_GLOBALS}/ThirdParty/snmp/snmp.jar"/>
 <property name="JarXerces"                  value="${DIR_GLOBALS}/ThirdParty/xerces-j/1.4.0/xerces-j.jar"/>
-<property name="JarJSON"                    value="${DIR_GLOBALS}/ThirdParty/json/json-20160212.jar"/>
+<property name="JarJSON"                    value="${DIR_GLOBALS}/ThirdParty/json/20160212/json-20160212.jar"/>
 
 <!-- XML-Beans -->
 <property name="JarXMLBeans"                value="${DIR_GLOBALS}/ThirdParty/XMLBeans/xmlbeans-2.5.0/lib/xbean.jar"/>
@@ -236,7 +236,7 @@
 <property name="JarMassaiSOAPSettings"      value="${JarDir}/massai-soap-settings.jar"/>
 <property name="JarMassaiAppAdapter"        value="${JarDir}/massai-app-adapter.jar"/>
 <property name="JarMassaiAdminApplication"  value="${JarDir}/massai-admin-app.jar"/>
-<property name="JarMassaiSSDMApplication"	value="${JarDir}/massai-ssdm-app.jar"/>
+<property name="JarMassaiSSDMApplication"   value="${JarDir}/massai-ssdm-app.jar"/>
 <property name="JarMassaiReleaseNotes"      value="${JarDir}/releasenotes.jar"/>
 <property name="JarMassaiStatechart"        value="${JarDir}/massai-statechart.jar"/>
 <property name="JarMatacClient"             value="${JarDir}/MATAC-CLIENT.jar"/>
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index 7ebb052..1b46bba 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -24,7 +24,7 @@
 
     <!-- Definition der Verzeichnisse mit den Java-Modulen ======= -->
     <property name="CommonDirList"   value="${CommonJavaDir}/OpenORB-1.3.1/OpenORB"/>
-    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/OPI_CCV20120201,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/ApplicationAdapter,${PlatformJavaDir}/AdminApplication,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/SOAP-Settings,${PlatformJavaDir}/CLAApplet"/>
+    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/OPI_CCV20120201,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/ApplicationAdapter,${PlatformJavaDir}/AdminApplication,${PlatformJavaDir}/SSDMApplication,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/SOAP-Settings,${PlatformJavaDir}/CLAApplet"/>
     <property name="AblsDirList"     value="${AblsJavaDir}/KioskProxy,${AblsJavaDir}/TunnelServer"/>
     <property name="AppDirList"      value=""/>
 
-- 
2.41.0.windows.1

